papers/whitepaper.md

```diff
@@ -111,3 +113,141 @@ The focus of the current text is to explore at a deeper level the Atoma's unique
+Notice that, in such case the request input data needs not be on-chain data, or either public data and it can be submitted through a frontend UI, a desktop application, a mobile application, similar to the `Externally owned request` case. In this case, the transaction only goes through if the required smart contract 'authorizes' it through the Atoma on-chain logic contract. 
+
+Nodes on the Atoma Network listen to blockchain events that are emitted once new request transactions are submitted to the Atoma on-chain logic contract. Once it listens to a new event, it can check if it has been selected to process the request, or not. If it has been selected, it should start processing the request right away. If the event contains all the available input data (in case, the transaction was submitted another smart contract in a `Smart contract authenticated request` case), or it must then fetch the input data from an external API call (in case, the transaction was submitted through a `Externally owned request` case) to the application the user is interacted with, so that it can start processing the request, right away.
+
+**Node collateral and rewards:** As already mentioned, nodes on the Atoma Network are required to deposit a given amount of collateral, upon registration. This collateral is subject to slashing in the cases of malicious behavior or in case the node becomes unresponsive (it stops processing requests for a long period of time). It was already mentioned that nodes are only able to withdraw that collateral in the epoch following de-registration. 
+
+If a node timeouts for a given request, a fixed percentage of its deposited collateral is slashed. Part of the slashed collateral goes to the user that initiated the request, to compensate the users for a poor service. Another part of the slashed collateral goes to the Atoma community treasury fund, and the remaining part is burned. This incentivizes nodes to be always responsive, otherwise they lose part of their stake in the network, while compensating both the user (with a refund), the network through treasury fund allocation and contributing for the TOMA token appreciation through deflation.
+
```

Comment:
If a node timeouts for a given request, a fixed percentage of its deposited collateral is slashed. Part of the slashed collateral goes to the user that initiated the request, to compensate them for a poor service. Another part of the slashed collateral goes to the Atoma community treasury fund, and the remaining part is burned. This incentivizes nodes to be always responsive, otherwise they lose part of their stake in the network while compensating both the user (with a refund), the network through treasury fund allocation, and also contributing to the TOMA token appreciation through deflation.